eos/rare-b-decays/nonlocal-formfactors-p-to-p.cc

```diff
@@ -61,6 +61,11 @@ namespace eos
                     return 0.0;
                 }
 
+                virtual complex<double> normalized_plus(const double & q2) const
```

Comment:
``normalized_plus`` sounds like H normalized to unity, rather than the ratio with the local form factor. Can we use ``ratio_plus`` instead?